src/main/java/com/commercetools/sync/inventory/utils/InventoryUpdateActionUtils.java

```diff
@@ -0,0 +1,106 @@
+package com.commercetools.sync.inventory.utils;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.inventory.commands.updateactions.ChangeQuantity;
+import io.sphere.sdk.inventory.commands.updateactions.SetExpectedDelivery;
+import io.sphere.sdk.inventory.commands.updateactions.SetRestockableInDays;
+import io.sphere.sdk.inventory.commands.updateactions.SetSupplyChannel;
+import io.sphere.sdk.models.Reference;
+
+import javax.annotation.Nonnull;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
+
+/**
+ * This class provides static utility methods for building update actions related to inventories.
+ */
+public final class InventoryUpdateActionUtils {
+
+    private InventoryUpdateActionUtils() {
+        throw new AssertionError();
+    }
+
+    /**
+     * Compares the {@code quantityOnStock} values of a {@link InventoryEntry} and {@link InventoryEntryDraft}
+     * and returns a {@link Optional} of update action, which would contain the {@code "changeQuantity"}
+     * {@link UpdateAction}. If quantity on stock from {@code newEntry} is {@code null} or no update action is needed,
+     * for example in case where both the {@link InventoryEntry} and the {@link InventoryEntryDraft} have the same
+     * {@code quantityOnStock} values, the empty optional would be returned.
+     *
+     * @param oldEntry the inventory entry that should be updated
+     * @param newEntry the inventory entry draft which contains new quantity on stock.
+     * @return optional containing update action or empty optional if quantities on stock are indentical or quantity on stock
+     * from {@code newEntry} is null.
+     */
+    @Nonnull
+    public static Optional<UpdateAction<InventoryEntry>> buildChangeQuantityAction(@Nonnull final InventoryEntry oldEntry,
+                                                                                   @Nonnull final InventoryEntryDraft newEntry) {
+        final Long newQuantityOnStock = newEntry.getQuantityOnStock();
+        if (newQuantityOnStock == null) {
```

Comment:
Updated. Now null with be treated like 0 (like empty stock) 86632b97c771af1f8db83e5a00fb9465ba334fa1